### PR TITLE
Upgrade `ava` and `meow`

### PR DESCRIPTION
**Pull request by AprilSylph**
Upgrading to `ava@latest` and `meow@v9` eliminates all npm vulnerability warnings. Somehow I was under the impression that there were none following merge of #47, probably due to my hesitance to commit the lockfile, which I have done here.

### CODE REVIEW

File: test/cli.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ test('Exits w/ message when param required by "Web Store Upload" is not provided
 
 test('It should not exit when --source param is not provided', async t => {
     try {
-        await execa('../index.js', ['upload'], {
+        await execa('./index.js', ['upload'], {
```
Comment (fregante) [inline test/cli.js:24]: Why was this changed? It should not load the file in the test folder but the one at the root

File: test/util.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import test from 'ava';
-import util from '../util';
-import stubs from './helpers/stubs';
+const test = require('ava');
+const util = require('../util');
+const stubs = require('./helpers/stubs');
```
Comment (fregante) [inline test/util.js:3]: These should not be changed but instead the the rest of the module should be turned into ESM (only).

File: test/cli.js (hunk 1 above)
Comment (AprilSylph) [reply test/cli.js:24]: All the file paths are resolved from the root now. Leaving them as-is makes `ava` complain about the fixtures directory not existing.

File: test/util.js (hunk 2 above)
Comment (AprilSylph) [reply test/util.js:3]: > move the AVA upgrade into a separate PR (because it means converting the module into ESM)

Given that (as far as I can tell, anyway) this does *work*, couldn't converting everything to ESM come later?

Comment (fregante) [reply test/util.js:3]: I'd rather not have to change everything to CJS and then back to ESM, but I think it's ok

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Cheers April!

**Comment by fregante** [comment]
Since this increases the minimum node version to v12 it will be released as a breaking release in a few days. Feel free to update any other dependencies before then.

**Comment by fregante** [comment]
- Done: https://github.com/fregante/chrome-webstore-upload-cli/pull/51

**Comment by fregante** [comment]
v2.0.0-0 (prerelease) has been published with all the latest updates. There are a few breaking ones:

- Node 12.20 or later
- The command is now called `chrome-webstore-upload`, not `webstore`

And also this is an important change, but it should still support any previously-generated tokens (which still require the secrets):

- https://github.com/fregante/chrome-webstore-upload/pull/46


It'd be great if you could test it as well. It must be installed with `npm install chrome-webstore-upload-cli@next`

**Comment by AprilSylph** [comment]
It seems that environment variables aren't automatically detected anymore.

**Comment by AprilSylph** [comment]
Yep, my existing keys seem to work just fine!
